labs/lab18-helm-charts/README.md

```diff
@@ -0,0 +1,435 @@
+   * **Replicated** - This option will eventually be depecrated and should not be selected. With this option, App Manager will first generate Kubernetes Manifests from the Helm Chart (i.e., run `helm template`) before they are deployed to the cluster as manifests.
+
+* Note how a new file is automatically created by Replicated that includes details about the chart we just added.
+
+<p align="center"><img src="img/helm-file.png" alt="helm file" width="600" margin=auto/></img></p>
+
+As a last step, save the release we've created 
+<p align="center"><img src="img/save release.png" alt="helm file" width="600" margin=auto/></img></p>
+
+### Using the CLI
+
+For the CLI, you should have already completed the [Hello World](https://github.com/replicatedhq/kots-field-labs/blob/main/labs/lab00-hello-world/README.md) Lab, as it covers how to set up your local Dev environment with the Replicated CLI.
+
+Make sure to update your environment variables to interact with this application. See [Get Started -> Steps 1 and 2](https://github.com/replicatedhq/kots-field-labs/blob/main/labs/lab00-hello-world/README.md)
+
+
+`REPLICATED_APP` should be set to the app slug from the Settings page. You should have received your App Name
+ahead of time.
+
+<p align="center"><img src="img/application-slug.png" alt="Application Slug" width="600" margin=auto/></img></p>
+
+
+`REPLICATED_API_TOKEN` should be set to the previously created user api token. See [Get Started -> Steps 1 and 2](https://github.com/replicatedhq/kots-field-labs/blob/main/labs/lab00-hello-world/README.md)
+
+Once you have the values, set them in your environment. If you are still in the `charts` directory, let's get back to the root of the directory we created earlier
+
+```bash
+    $ cd ../../..
+```
+Now to set your environment variables:
+
+```bash
+    $ export REPLICATED_APP=...
+    $ export REPLICATED_API_TOKEN=...
+```
+
+We need a directory that will contain the Helm Chart and the Replicated manifests. Let's create a directory for our application, and in there a `manifests` directory that will contain the manifests and helm chart.
+
+```bash
+   $ mkdir helm-wordpress
+   $ cd helm-wordpress
+   $ mkdir manifests
+```
+
+We need to move the helm chart package we created previously to the manifests directory we just created. Depending on where your helm chart was created, the path in the example command below may be different.
+
+```bash
+    $ mv ../charts/bitnami/wordpress/wordpress-14.0.7.tgz manifests
+```
+
+Next, let's create a couple of manifests to finish our first release. Both of these manifests are Replicated [Custom Resources](https://docs.replicated.com/reference/custom-resource-about) and should be created in the `manifests` directory.
+
+#### Adding kots-app.yaml
+
+The first Custom Resource we'll create is [Application](https://docs.replicated.com/reference/custom-resource-application). This manifest includes details about the application we created.
+
+``` yaml
+#kots-app.yaml
```

Comment:
```suggestion
cat <<EOF > manifests/kots-app.yaml
```

Reply:
@jdewinne @cremerfc  what do you think about a heredoc instead of "make this file however you want". ?

Reply:
I'm ok with either. If you came this far through the labs, and you know Helm, I'm going to assume you can also create file in your favorite editor of choice.

Reply:
okay we can leave for now, easy to change later. I have proposed a few things in the style guide as far as consolidating on one way or another